examples/notebooks/Makefile

```diff
@@ -0,0 +1,52 @@
+# This Makefile runs notebooks and converts them into a different output format.
+# The notebook execution is a two-step process:
+#
+# 1. Run then notebook and capture outputs in an *.ipynb file.
+# 2. Translate the *.ipynb file to Markdown
+#
+# Ideally, we would go directly to the Markdown output, but jupytext only
+# captures outputs when converting to *.ipynb. So we use nbconvert afterwards to
+# translate the *.ipynb files to Markdown.
+
+VENV := venv
+BUILD := output
+NOTEBOOKS = $(wildcard *.md)
+MD = $(addprefix $(BUILD)/md/, $(NOTEBOOKS))
+IPYNB = $(addprefix $(BUILD)/ipynb/, $(NOTEBOOKS:.md=.ipynb))
+
+all: md ipynb
```

Comment:
not a make expert, so I might be confused, but it seems that the call to `ipynb` here is not necessary as `md` already has a dependency on `ipynb` (at `$(BUILD)/md/%.md: $(BUILD)/ipynb/%.ipynb`)